botbot/apps/plugins/runner.py

```diff
@@ -187,11 +204,27 @@ def register(self, plugin):
                 continue
             if (not key.startswith('__') and
                     getattr(attr, 'route_rule', None)):
-                LOG.info('Route: %s.%s listens to %s for matches to %s',
-                         plugin.slug, key, attr.route_rule[0],
-                         attr.route_rule[1])
-                getattr(self, attr.route_rule[0] + '_router').setdefault(
-                    plugin.slug, []).append((attr.route_rule[1], attr, plugin))
+                router = attr.route_rule[0]
+                if router == "commands":
+                    LOG.info('Route: %s.%s listens to %s for command %s',
+                             plugin.slug, key, attr.route_rule[0],
```

Comment:
this can be replaced with router as well.

Also make a variable for ```attr.route_rule[1]``` and replace that as well